ONSdigital/dp-frontend-renderer
# Feature/geography area page update

### What

Add the next step in the /geography journey, so that the user can see details of an area and what datasets are related to it. 

### How to review

1. Make sure you have a the [code list imported](https://github.com/ONSdigital/dp-code-list-scripts#import-to-a-new-development-environment) into Neo4j.
2. Switch to feature/geography-area-page-update branch on the [frontend renderer](https://github.com/ONSdigital/dp-frontend-renderer/pull/246) and [frontend geography controller](https://github.com/ONSdigital/dp-frontend-geography-controller/pull/9).
3. You should only need to run: Neo4j, Kafka, dp-frontend-geography-controller, dp-frontend-renderer, dp-frontend-router and the dp-code-list-api.
4. Go to /geography through the router, you should see the geography homepage.
5. Click on 'Local authorities' and you should be taken to a page that lists the 391 local authorities.
6. Click on an area and check area page renders 

### Who can review

Anyone.
